### DISCUSSION TITLE
Quiz#2646

### DISCUSSION THREAD

**Discussion by sorn32** [1101-GS4719-C/Python-Community-GS4719-C#1675] [General]
在try except例外處理的第2646題，答案應該是C? 但提交每一個選項都寫錯誤இдஇ
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/76047120/6dd4ed6f-7dfa-49c4-bc4c-554e002cad59)

**Comment by BOSENCHANG2004** [comment]
正確答案應該是 C。
 A 選項：使用 finally: 但沒有 except:
finally 是不論有沒有發生錯誤都會執行的區塊，不能單獨用來捕捉錯誤。正確應為 try...except...finally 才對。
B 選項：使用 catch:
catch 是 Java / C++ 的用法，Python 中沒有 catch，應該是 except。
D 選項：使用 handle:
Python 沒有 handle 這個關鍵字來處理例外，所以這也是錯誤語法。
可能是題目設定有錯誤
